app/models/enrollment/api_entreprise.rb

```diff
@@ -0,0 +1,3 @@
+class Enrollment::ApiEntreprise < Enrollment
+  resourcify
```

Comment:
this instruction is not needed anymore (I removed rolify in #107). I can remove it for you if you want.